packages/runner-ct/src/app/app.jsx

```diff
@@ -1,169 +1,61 @@
-import cs from 'classnames'
-import { observable } from 'mobx'
 import { observer } from 'mobx-react'
 import PropTypes from 'prop-types'
-import React, { Component } from 'react'
-import { findDOMNode } from 'react-dom'
+import React from 'react'
```

Comment:
would it be too time consuming to make this into a `tsx` file too?  if not, I think we should go for it.

I think the time investment now to TS all the things will be worth it.

Reply:
I will be happy to do this in separate PR